### PR TITLE
Jitsu

### PR DESCRIPTION
**Pull request by tony0306**
## Changes on the Jitsu exposed ports
Adding port 8001 to have successful liveness and readiness probes 

## Checklist

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: jitsu/helm/jitsu/templates/deployment.yaml [hunk 1]
```diff
@@ -43,6 +43,9 @@ spec:
             - name: config
               containerPort: 8000
               protocol: TCP
+            - name: test
```
Comment (michaeljguarino) [inline jitsu/helm/jitsu/templates/deployment.yaml:46]: is `test` the right name for the port or does it serve a different role within jitsu?

Comment (tony0306) [reply jitsu/helm/jitsu/templates/deployment.yaml:46]: I think a better name will be readiness/liveness. I used "test" as this is used by the probes to initially test the url. As this was failing (port was not exposed), k8 was killing the deployment.

Comment (michaeljguarino) [reply jitsu/helm/jitsu/templates/deployment.yaml:46]: how about `health`?

Comment (tony0306) [reply jitsu/helm/jitsu/templates/deployment.yaml]: I changed to health. I was also supposed to change it in here:
          livenessProbe:
            httpGet:
              path: /
              port: health

but on the deployment I'm getting this error. 
Liveness probe errored: strconv.Atoi: parsing "health": invalid syntax

Reverted to http.

File: jitsu/helm/jitsu/templates/service.yaml [hunk 2]
```diff
@@ -15,5 +15,9 @@ spec:
       targetPort: config
       protocol: TCP
       name: config
+    - port: 8001
+      targetPort: 8001
```
Comment (DavidSpek) [inline jitsu/helm/jitsu/templates/service.yaml:19]: This should probably be set to `health`

**Review by michaeljguarino — APPROVED** [review]
rm the file for n8n but otherwise I think this is good to go

**Pull request merged**